Gemfile

```
@@ -81,6 +84,10 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  # Allow us to automate various tasks whenever file or directories are modified during development
+  gem 'guard'
+  # Allow us to automatically & intelligently launch specs when files are modified
+  gem 'guard-rspec', require: false
```

Test Suggested change

```suggestion
  gem 'guard-rspec', require: false, test: true
```

---

```suggestion
  gem 'guard-rspec', require: nil
```

```suggestion
  gem 'guard-rspec', require: true
```